src/client/packaging/npm/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "delphi_epidata",
-  "version": "0.4.4",
+  "version": "4.1.5",
```

Comment:
possibly/probably unrelated to this change, but why doesnt `package-lock.json` get updated during the standard `delphi-epidata` new version release procedure?  the associated `package.json` does...  i guess it doesnt matter